Findingcircles.py

```diff
@@ -0,0 +1,371 @@
+# This file forms the backbone for the alternative analysis of "donuts"
+# from AuxTel. It is based on work initially done by Chris Stubbs.
+import cv2
+import lsst.rapid.analysis.butlerUtils as butlerUtils
+import matplotlib.pyplot as plt
+import numpy as np
+import numpy.ma as ma
+from lsst.rapid.analysis.imageExaminer import ImageExaminer
+import os
+
+
+def findCircles(day_obs, seq_nums, doPlot=False, planeSkew=False, config=None, path=None,
+                butler_type='NCSA', **kwargs):
+    """Let's find all the circles! this function simply loops over a
+    sequence list (seq_List) the function FindCircle, which does all
+    the work for a single exposure.
+
+    Parameters
+    ----------
+    day_obs : `string`
+        The observation date we are looking to find circles for.
+
+    seq_nums : `iterable`
+        The iterable of sequences we are hoping to obtain exposures for.
```

Comment:
I think I will work on a class based version of this, but seeing as it is in no rush it might be a winter break project.

Reply:
I more meant that if, instead of taking a single dayObs and a list of seqNums, it took a single list of dataId dicts, then it would Just Work. Do you see what I mean?

Reply:
I see what you mean, and was saying it will be something I will try and incorporate in a future version.

Reply:
Ah OK, I see what you mean.

Reply:
Maybe just stick `TODO` comments in the code so that you've got notes on what to do if you come to do those.

Reply:
That is a good point, will do!